landing-page/content/common/contribute.md

```diff
@@ -83,6 +83,69 @@ Follow the instructions for [Eclipse](https://github.com/google/google-java-form
 [IntelliJ](https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides) to install the **google-java-format** plugin (note the required manual actions for IntelliJ).
 
 
+## Semantic Versioning
+
+Apache Iceberg leverages [semantic versioning](https://semver.org/#semantic-versioning-200) to ensure compatibility
+for developers and users of the iceberg libraries as APIs and implementations evolve.  The requirements and
+guarantees provided depend on the subproject as described below:
+
+### Major Version Deprecations Required
+
+__Modules__
+`iceberg-api`
+
+The API subproject is the main interface for developers and users of the Iceberg API and therefore has the strongest
+guarantees.  Evolution of the interfaces in this subproject are enforced by [Revapi](https://revapi.org/) and require
```

Comment:
nit: extra space here `guarantees.  Evolution`

Can we just use newline for each sentence? That helps making future changes easier to minimize line changes.

Reply:
good point, done